docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/dynamic-data-api-integration.md

```diff
@@ -136,6 +136,108 @@ class DynamicEntityBackendApiConfig extends SprykerDynamicEntityBackendApiConfig
 }
 ```
 
+### Dynamic Data Install Configuration
+
+4. It is possible to install default configuration data by creating a configuration file:
```

Comment:
Don't use "possible" or "impossible" to mean "you can" or "you can't".

If this step is mandatory, the apply this change:
```suggestion
1. Install default configuration data: create a configuration file:
```
If the step is optional, apply this change:
```suggestion
1. Optional: To install default configuration data, create a configuration file:
```